pkg/client/client.go

```diff
@@ -1486,7 +1486,7 @@ func (c *KpmClient) DownloadFromOci(dep *downloader.Oci, localPath string) (stri
 }
 
 // PullFromOci will pull a kcl package from oci registry and unpack it.
-func (c *KpmClient) PullFromOci(localPath, source, tag string) error {
+func (c *KpmClient) PullFromOci(localPath, source, tag string, skipTLSVerify bool) error {
```

Comment:
Yes this is failing because there is another function CompileOciPkg that uses the same function we made changes to, do you have an idea how can I handle the tls flag argument for that func?